Tema4/src/Order_Product.java

```diff
@@ -0,0 +1,172 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * @author Bogdan
+ */
+
+/**
+ * This is a Local Store Simulation based on JAVA OOP Implementation.
+ * Based on the user input he/ her can input
+ * The store has a list of items in stock and each item refers to a product.
+ *A product is identified uniquely by ID.
+ */
+
+public class Order_Product {
+    public static void main(String[] args) {
```

Comment:
You created the menu as specified but it doesn't actually do anything useful. I think you tried to make it work with the menu.... then misunderstood this :
"Important note:
- if you don't manage to create and work with the program menu described above, just create programmatically (write the Java code to instantiate) a list of products of various types; then place a few orders; then display the daily report for a particular date."

And you created main methods that would create lists in the other classes.

What we asked there is that you:
- create all the classes including the store and making them work
- to make the store work you would need some attributes List<Product> and List<Order> . You also needed methods for adding a product, selling a product and displaying orders.
- here is the difference: IF you cannot work with the Scanners and System.outs and with the menu, THEN we would accept that you create a main method somewhere and create some interractions with the Store class that would simulate human interraction. So instead of asking the user to insert the product details you would just create the Product with dummy values and pass that product to the store. Instead of asking the user which product to sell, you would directly say a dummy ID to the store to sell (in your code in your main function). So a simulation of human interraction.